### DISCUSSION TITLE
The snapshot is missing. Displaying current state of the DOM

### DISCUSSION THREAD

**Discussion by oaktree4** [cypress-io/cypress#24830] [Questions and Help]
Hello everyone, getting into cypress and stumbling upon the following issue.

Im tryng to run this simple test (visit a webapp)

describe('empty spec', () => {
    it('Test', () => {
      cy.visit('https://test.app.sequello.com/')
      
    })
  })

Although, what happens in cypress is the following. The screen is blank and when I hover over the test body I get the following error message "The snapshot is missing. Displaying current state of the DOM"

![image](https://user-images.githubusercontent.com/63807087/204152735-de7a0540-9866-4bf4-9a94-9b33ceb95cd6.png)

Can someone tell me what is the issue here and how can I solve it?

Kind regards

**Comment by samelawrence** [comment]
I also have this same issue, in fact - between crashes and missing snapshots, for the past 5 years, I have *never* been able to properly use Cypress snapshots for local testing. The closest I can get is Test Replay in the Dashboard. I would really love to be able to use this feature of Cypress as it is intended. It seems like a memory leak has possibly been fixed recently? My test runner always used to crash if the `numTestsKeptInMemory` value was set to anything greater than `0`. That no longer seems to happen, but even setting the value back to the default of `50`, I always get the 'Snapshot is missing' pop-up.
